src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static aliveAnimals() {
+    this.alive = this.alive.filter(animal => animal.health > 0);
+  }
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
there must be just alive animals, so you don't need to create static aliveAnimals that returns filtered alive arr